### PR TITLE
Update hero and fix typo on Financial Well-Being Scale app

### PR DESCRIPTION
**Pull request by Scotchester**
A couple little things to make the testing prototype slightly more polished.

## Additions

- Proper hero illustration

## Changes

- Typo in page title fixed

## Testing

1. Pull branch
2. Run server
3. Visit http://localhost:8000/consumer-tools/financial-well-being/
4. See new hero illustration and grammatically correct `<title>`

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/wellbeing/jinja2/wellbeing/home.html [hunk 1]
```diff
@@ -12,35 +12,16 @@
...
+            background-image: url( '{{ static("img/fwb_hero_760x300.png") }}' );
             filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
-                src='/f/images/parents_hero_760x364.original.png',
-                sizingMethod='scale'
+                    src='{{ static("img/fwb_hero_760x300.png") }}',
+                    sizingMethod='scale'
             );
             padding-bottom: 41.4893617%;
         }
-        @media screen and ( min-width: 37.5625em ) {
```
Comment (sebworks) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: why are you removing this?

Comment (Scotchester) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: It was copied from typical Wagtail hero output when I originally put it in there. In the case of a hero like this one, though, where the small screen image is the same as the large screen image, it serves no purpose.

Comment (sebworks) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: why do you need the` padding-bottom` then?

Comment (Scotchester) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: Hmm... not 100% sure. Let me test again to be sure.

Comment (Scotchester) [inline cfgov/wellbeing/jinja2/wellbeing/home.html]: The padding bottom controls the size of the image box, and by extension, the size of the image itself (which is a background image with `background-size: contain`). It's calculated when Wagtail outputs the page, so your comment made me realize that I needed to update it to get the correct intended size for this image. Fix pushed!

**Pull request merged**